provider/resource_keycloak_openid_client_test.go

```diff
@@ -256,6 +256,38 @@ func TestAccKeycloakOpenidClient_ClientTimeouts_basic(t *testing.T) {
 	})
 }
 
+func TestAccKeycloakOpenidClient_Device_basic(t *testing.T) {
+	t.Parallel()
+	clientId := acctest.RandomWithPrefix("tf-acc")
+
+	oauth2DeviceCodeLifespan := "300"
```

Comment:
since we're handling this as a duration string, I expected this to have a different format
```suggestion
	oauth2DeviceCodeLifespan := "300s"
```

Reply:
The client settings are using for every time based option just a string without a unit.

Examples in other test cases: 

`TestAccKeycloakOpenidClient_AccessToken_basic`:
- `accessTokenLifespan := "1800"`

`TestAccKeycloakOpenidClient_ClientTimeouts_basic`:
- `offlineSessionMaxLifespan := "1900"`
- `sessionMaxLifespan := "2100"`

Reply:
My bad, I got this confused with the `keycloak_realm` attributes, which are duration strings.